tests/unit_tests/t_opt_parser.rb

```diff
@@ -62,6 +62,26 @@ def test_full_option_import
     option_ass(options)
   end
 
+  def test_default_cachehttp
+    opp2 = OptParser.new
+    full_hash = { repo: 'gino/gitarro', context: 'python-t',
+                  test_file: 'gino.sh',
+                  git_dir: 'gitty' }
+    opp2.options = full_hash
+    options = opp2.cmdline_options
+    assert_equal('/tmp', options[:cachehttp])
+  end
+
+  def test_custom_cachehttp
+    opp2 = OptParser.new
+    full_hash = { repo: 'gino/gitarro', context: 'python-t',
+                  test_file: 'gino.sh', cachehttp: '/tmp/customd',
```

Comment:
customd?

Reply:
it is just a test, for saying that the name is custom :rofl: